spec/support/helpers/contract_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Helpers
+  module ContractHelpers
+    module_function
```

Comment:
Please describe where you have planed to use `ContractHelpers.expect_errors()`???

Reply:
I guess never, I'll remove it.